src/transforms/remap.rs

```diff
@@ -64,14 +104,44 @@ impl TransformConfig for RemapConfig {
         Input::any()
     }
 
-    fn outputs(&self) -> Vec<Output> {
+    fn outputs(&self, merged_definition: &schema::Definition) -> Vec<Output> {
+        // We need to compile the VRL program in order to know the schema definition outpuf of this
+        // transform. We ignore any compilation errors, as those are caught by the transform build
+        // step.
+        //
+        // TODO: Keep track of semantic meaning for fields.
```

Comment:
This will happen in a follow-up PR, once I land support in VRL to modify semantic meaning of fields.